srcalc/src/main/java/gov/va/med/srcalc/vista/HealthFactor.java

```diff
@@ -0,0 +1,63 @@
+package gov.va.med.srcalc.vista;
+
+import java.util.Date;
+import java.util.Objects;
+
+/**
+ * A class that will store the name of a health factor along with the 
+ * date on which that factor was noted. Instances of this class are immutable.
+ */
+public class HealthFactor
```

Comment:
Other than my minor comments, I think this class is useful and simple. Good job.